trade-finance-manager-ui/server/routes/auth/configs/unauthenticatedAuthSso.ts

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+import { GetRouter } from '../../../types/get-router';
+import { UnauthenticatedAuthController } from '../../../controllers/auth/authSso/unauthenticated-auth.controller';
+
```

Comment:
Can we rename the file to `unauthenticated-auth-sso` please

Reply:
updated